<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CPointNameTable</class>
 <widget class="QDialog" name="CPointNameTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>点名选择</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>选择点名：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="editPointName"/>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonEnter">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeWidget" name="treePointName">
     <property name="styleSheet">
      <string notr="true">font: 75 16pt &quot;仿宋&quot;;</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonCancel</sender>
   <signal>clicked()</signal>
   <receiver>CPointNameTable</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>418</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
